packages/formation/js/accordion.js

```diff
@@ -83,9 +83,10 @@ const addAccordionClickHandler = () => {
         // and if it is a .usa-accordion-button.
         // Specifically React Components.
         if (accordionButton && !accordionButton.onclick) {
-          const multiSelectable = toBoolean(
-            element.getAttribute('aria-multiselectable'),
-          );
+          const multiSelectable =
+            // Don't use aria-multiselectable, it's not a valid use of that attribute
+            toBoolean(element.getAttribute('aria-multiselectable')) ||
```

Comment:
Do you have any interest in adding a `console.warn()` if `aria-multiselectable` is found?

Reply:
That's a good idea, I'll add that.